### PR TITLE
ssd1306: add DrawBitmap() function to complete Displayer interface

### PR DESCRIPTION
**Pull request by deadprogram**
This PR adds a `DrawBitmap()` function to the `ssd1306` driver in order to complete the `Displayer` interface as needed for the Thumby to be added to the `board` package.

### CODE REVIEW

**Review by sago35 — APPROVED** [review]
LGTM

File: ssd1306/ssd1306.go [hunk 1]
```diff
@@ -338,3 +343,22 @@ func (b *SPIBus) tx(data []byte, isCommand bool) error {
...
+	if x < 0 || x+int16(width) > d.width || y < 0 || y+int16(height) > d.height {
+		return errOutOfRange
+	}
+
+	zeroColor := color.RGBA{0, 0, 0, 255}
+	for i := 0; i < width; i++ {
+		for j := 0; j < height; j++ {
+			if bitmap.Get(i, j).RGBA() != zeroColor {
+				d.SetPixel(x+int16(i), y+int16(j), color.RGBA{255, 255, 255, 255})
+			}
```
Comment (aykevl) [inline ssd1306/ssd1306.go]: You are only changing dark pixels to light pixels, but make it impossible to change light pixels to dark? Why is that?

This makes a lot more sense to me:

```suggestion
			d.SetPixel(x+int16(i), y+int16(j), bitmap.Get(i, j).RGBA())
```

Comment (deadprogram) [reply ssd1306/ssd1306.go]: That looks a lot better @aykevl thanks. Change made.

**Comment by deadprogram** [comment]
Thanks for the feedback and reviews @aykevl and @sago35 now merging.

**Pull request merged**